src/transformer/section-category.ts

```diff
@@ -168,7 +174,7 @@ export const buildSectionCategory = (
       return 'MPSectionCategory:abstract-graphical'
 
     default:
-      return node.attrs.category || undefined
+      return node.attrs.category || node.attrs.group || undefined
```

Comment:
Why is this needed?

Reply:
This was left unintentionally when removing the section_container node.